cadre.go

```diff
@@ -151,16 +151,21 @@ func (c *cadre) startGRPC() {
 		t := time.NewTicker(5 * time.Second)
 
 		for {
+			// Nothing to check
+			if c.grpcHealthService == nil {
```

Comment:
Can be moved outside the closure - there's no point starting a goroutine only to return from it.

Reply:
changed and moved to separate function

Reply:
changed and moved to separate function